batavia/types/List.js

```diff
@@ -192,7 +192,11 @@ batavia.types.List = function() {
 
             return new List(Array.prototype.slice.call(this, start, stop));
         } else {
-            throw new batavia.builtins.TypeError("list indices must be integers, not " + batavia.type_name(index));
+            var msg = "list indices must be integers or slices, not ";
+            if (batavia.BATAVIA_MAGIC == batavia.BATAVIA_MAGIC_34) {
```

Comment:
according to https://github.com/pybee/batavia/blob/master/tests/utils.py#L398 in some places we simply swap out the error message in Python during tests so that assertion passes. given that, Im not sure if this approach makes sense. this approach matches error message to of corresponding Python version whereas the other approach is a bit simpler.

@freakboy3742 thoughts?